[ZH] Add Project Loved (#4640)
ppy/osu-wiki: wiki/Project_Loved/zh.md

## 使一张谱面进入Loved状态

谱面将由在经过一个提名和投票程序后进入Loved状态。

Project Loved Team团队的**队长**会定期为一个特定游戏模式的谱面提名。这些谱面集几乎都来自osu!社区的提交。当前的Loved提交表格和列表可以在以下的链接找到：

- [osu!/osu!catch 提交表格](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)
- [osu!taiko 提交表格](https://docs.google.com/forms/d/e/1FAIpQLSclPWyjFByhHP45AtKD49y0RSl1TK5UOzD4dVdvjfJJQ1aCXQ/viewform)
- [osu!mania 提交表格](https://docs.google.com/forms/d/e/1FAIpQLSeaGfoQNGMqw4qQcqRPItUZILh2fGwJR6ly6cZNY9OWPXkFhw/viewform)
- [osu!/osu!catch 列表](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit)
- [osu!taiko 列表](https://docs.google.com/spreadsheets/d/1Nk96z_xat_7ypsDF1sCTDO4i_CnHarcrCbGoTmgwHHE/edit)
- [osu!mania 列表](https://docs.google.com/spreadsheets/d/1sjkTwUSvQ5Me-6rK61rToTg2bU-yX9X29CXdzttvhtM/edit)

*注意: 如果你访问表格时受到了限制，你可以联系任何[Project Loved Team 成员](/wiki/People/The_Team/Project_Loved_Team#team-members)来替你提交谱面*

被提名的谱面会被发送到[Project Loved 论坛](https://osu.ppy.sh/community/forums/120)，社区成员可以投票决定他们是否想要这个地图集被Loved。在投票结束时，如果地图的“Yes”票数达到指定的百分比，就会进入Loved状态。

### 要求

> 谱面需要满足一些很小的规定以使它可以被Loved提名。队长认为谱面集需要符合以下条件:

Reviewer: ```suggestion
谱面需要满足一些要求才能被提名为Loved，队长会考虑以下条件:
```

Author: 很小的规定是原文的说法，我觉得就算不用很小的也应该用点别的词修饰？

Reviewer: 我认为 minimal 只是说“要求很低”，不应该翻译为“很小的”。

Author: 那就改成很低的要求好了

- 不是[Qualified](/wiki/Beatmaps#qualified), [Ranked](/wiki/Beatmaps#ranked)或者[Approved](/wiki/Beatmaps#approved)状态。
- 谱面制作者不反对谱面被Loved。
- 谱面制作者未被封禁或者限制。
- 至少有一个难度可以通过。
- 谱面至少要30个收藏（仅应用于osu!standard谱面）

> 然而，尽管几乎所有的[Pending和Working-in-progress](/wiki/Beatmaps#work-in-progress-and-pending)地图都符合这些标准，但由于g更选择性的提名和投票要求，很少有地图被投票到受欢迎的类别中。

Reviewer: ```suggestion
虽然几乎所有的[Pending和Working-in-progress](/wiki/Beatmaps#work-in-progress-and-pending)地图都符合这些标准，但出于更挑剔的提名条件与投票要求，只有少数谱面才能进入Loved。
```

Author: ？这里竟然有个字母落了，一会就改

进入Loved状态的谱面需要遵循一些额外的规则:

- [在Ranking Criteria的元数据部分](/wiki/Ranking_Criteria#metadata)
- [歌曲内容规定](/wiki/Rules/Song_Content_Rules)
- [视觉内容注意事项](/wiki/Rules/Visual_Content_Considerations)

与这些规则有关的任何问题通常在地图进行投票时得到解决。

### 提名优先级

对于队长提名的地图并没有严格的规定，但通常来说，提名的地图都是非常知名和被社区所喜爱的（正如标题的“Loved”）。有时候，如果谱面集比其他的要老得多，或者具有独特和高质量作图风格的谱面，那么它也会是首选。

## 更多信息

> 所有Project Loved的组织和计划都在[osu!dev Discord server](https://discord.gg/ppy)的`#osu-loved`频道。也有个位于[osu! Community Discord server](https://discord.gg/0Vxo9AsejDkGlk3H)的`#project-loved`频道。如果有任何问题、担忧或建议都可以到那里提出。

Reviewer: ```suggestion
所有Project Loved的组织和计划都在[osu!dev Discord server](https://discord.gg/ppy)的`#osu-loved`频道进行，此外还有一个位于[osu! Community Discord server](https://discord.gg/0Vxo9AsejDkGlk3H)的`#project-loved`频道。如果有任何疑虑或建议都可以到那里提出。
```
